docker/lr-jupyter/runJupyterNotebook.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
WDL tasks are run in docker containers. Is this saying, we are running docker in docker?

Reply:
Move this to scripts dir.